### PR TITLE
new configuration for HTML tags

### PR DESCRIPTION
**Pull request by SaulLu**
This PR proposes to change the configurations for the HTML tags and also for the entities (cc @manandey )

### CODE REVIEW

File: bsmetadata/hydra_configs/v2.yaml [hunk 1]
```diff
@@ -27,8 +27,8 @@ data_config:
     metadata_sep: ' | '
     metadata_key_value_sep: ': '
     metadata_probability: 1.0
-    treat_local_metadata_as_regular_text: false
-    add_local_metadata_special_tokens_in_prefix: false
+    treat_local_metadata_as_regular_text: true
```
Comment (SaulLu) [inline bsmetadata/hydra_configs/v2.yaml:30]: If I recall correctly this argument is used to know if we want to take the local metadata occurrences into account in the loss: I think that we want them to be taken into account.

Do you agree @manandey , @cccntu ?

File: bsmetadata/hydra_configs/v2.yaml [hunk 2]
```diff
@@ -27,8 +27,8 @@ data_config:
...
+    add_local_metadata_special_tokens_in_prefix: true
```
Comment (SaulLu) [inline bsmetadata/hydra_configs/v2.yaml:31]: I would suggest adding a parameter for the local metadata as well to allow their controllability afterwards.
A prompt beginning with `html |||` would triggers the addition of html tags inside afterwards. If the prompt doesn't begin with `html |||`, then we shouldn't have html tags in the text generated

File: bsmetadata/hydra_configs/v2.yaml [hunk 3]
```diff
@@ -37,33 +37,15 @@ data_config:
...
-        - caption
-        - colgroup
-        - thead
-        - tfoot
-        - tbody
-        - li
-        - ol
-        - ul
-        - span
+        tags_exceptions_to_txt_max_min_chr_len: null
```
Comment (SaulLu) [inline bsmetadata/hydra_configs/v2.yaml:42]: I'm not entirely sure about the yaml syntax here, but the idea is that I don't want to define exceptions (so in the code this should result in an empty list I believe)

File: bsmetadata/hydra_configs/v2.yaml (hunk 1 above)
Comment (manandey) [reply bsmetadata/hydra_configs/v2.yaml:30]: Yeah, I completely agree! :)

**Review by manandey — APPROVED** [review]
Thanks, Lucile!

File: bsmetadata/hydra_configs/v2.yaml (hunk 1 above)
Comment (cccntu) [reply bsmetadata/hydra_configs/v2.yaml:30]: Yes, I think it makes more sense to calculate the loss of html, too.

File: bsmetadata/hydra_configs/v2.yaml (hunk 2 above)
Comment (cccntu) [reply bsmetadata/hydra_configs/v2.yaml:31]: > A prompt beginning with html ||| would triggers the addition of html tags inside afterwards. If the prompt doesn't begin with html |||, then we shouldn't have html tags in the text generated

Sound good to me!

> I would suggest adding a parameter for the local metadata as well to allow their controllability afterwards.

I don't understand this. Does this change already does what you want? Or are you suggesting to add a different parameter?

Comment (SaulLu) [reply bsmetadata/hydra_configs/v2.yaml:31]: > Does this change already does what you want? 

yes exactly! My comment applies to all local-type metadata (for a long time the entities were also local-type metadata, I don't know what is the status currently). The idea is exactly what I explained in the following sentences: `html` is the parameter for the local metadata "html" :smile: 

Don't hesitate to tell if this is still confusing :smile:

Comment (manandey) [reply bsmetadata/hydra_configs/v2.yaml:31]: > for a long time the entities were also local-type metadata, I don't know what is the status currently

Entities are still local-type metadata. :slightly_smiling_face:

**Pull request merged**